examples/conversion/conversionExamples.m

```diff
@@ -0,0 +1,71 @@
+%% Convert r1 struct to projectClass.
+% 
+% The following example shows how to convert r1 structs into projectClass.
+% 
+% r1 struct is converted into a projectClass using the r1ToProjectClass function.
+% 
+% The functions takes the name of the r1 struct .mat file.
```

Comment:
This should be in the function docstring with the example usage

Reply:
Moved.